mscore/playpanel.cpp

```diff
@@ -60,6 +60,7 @@ PlayPanel::PlayPanel(QWidget* parent)
       volLabel();
       volSpinBox->setRange(-80,0);
       volSpinBox->setValue(-40.0);
+      volSpinBox->setDecimals(1);
```

Comment:
Might better be sone in the ui file

Reply:
OK. Makes sense. And, in fact, it was already set to 1 in UI file, so I've just removed the redundant line of code. Thanks.